src/test/java/org/elasticsearch/gateway/local/RecoveryBackwardsCompatibilityTests.java

```diff
@@ -77,7 +77,8 @@ public void testReusePeerRecovery() throws Exception {
         indexRandom(true, builders);
         ensureGreen();
         if (randomBoolean()) { // just make sure it doesn't break anything - we seal before we actually bump replicas
-            backwardsCluster().internalCluster().client().admin().indices().prepareSealIndices("test").get();
+            logger.info("--> trying to sync flush");
+            assertEquals(SyncedFlushUtil.attemptSyncedFlush(internalCluster(), "test").failedShards(), 0);
```

Comment:
yes, should be backwardsCluster()... we always have current version nodes in the test unless annotated otherwise. is that what you mean?